suite-native/module-send/src/screens/SendOutputsScreen.tsx

```diff
@@ -45,7 +45,7 @@ export const SendFormScreen = ({
                         />
                     </HStack>
                 </VStack>
-                <SendForm accountKey={accountKey} />
+                <SendOutputsForm accountKey={accountKey} />
```

Comment:
I believe it should either be as it was. We're not sending outputs per say, we're just sending transaction which has outputs, no? This name is pretty confusing IMO

Reply:
It has to be like this. Because in the next PR will be added `SendFeesForm`.